pipeline/tests/pil.rs

```diff
@@ -123,7 +123,7 @@ fn test_external_witgen_both_provided() {
 }
 
 #[test]
-#[should_panic = "called `Result::unwrap()` on an `Err` value: Generic(\"main.b = (main.a + 1);:\\n    Linear constraint is not satisfiable: 18446744069414584320 != 0\")"]
+#[should_panic = "Witness generation failed."]
```

Comment:
Error message changed, because it used to fail on the last row when solving for the partial first row already (where we `unwrap()`). Now, the identities without next reference are skipped, so it only fails during the "normal" witness generation.